apps/examples/app.json

```diff
@@ -6,14 +6,13 @@
     "orientation": "portrait",
     "icon": "./assets/icon.png",
     "userInterfaceStyle": "light",
+    "newArchEnabled": true,
```

Comment:
https://docs.expo.dev/guides/new-architecture/#enable-the-new-architecture-in-an-existing-project